paper.md

```diff
@@ -94,7 +94,7 @@ and improvements to the generation process can operate solely on this universal
 Currently Hypothesis uses this format to support two major use cases:
 
 1. It is the basis of its approach to test-case reduction, allowing it to support more powerful test-case reduction than is found in most property-based testing libraries with no user intervention.
-2. It supports Targeted Property-Based Testing [@DBLP:conf/issta/LoscherS17], which uses a score to guide testing towards a particular goal (e.g. maximising an error term). In the original implementation this would require custom mutation operators per test,
+2. It supports Targeted Property-Based Testing [@DBLP:conf/issta/LoscherS17], which uses a score to guide testing towards a particular goal (e.g., maximising an error term). In the original implementation this would require custom mutation operators per test,
```

Comment:
Apparently it had completely passed me by that US English does this. I'm happy to go along with it I guess, but what a bizarre convention.

Reply:
e.g. is for example, and if was written out as such, it would get a comma in US English, so the abbreviation does too